libs/blocks/hero-marquee/hero-marquee.css

```
@@ -396,6 +439,32 @@ html[dir="rtl"] .hero-marquee li.icon-item span.icon {
   }
 }
 
+/* mweb mobile defaults */
+.hero-marquee.mobile-left-align.center {
+  text-align: left;
+  align-items: flex-start;
+}
+
+.hero-marquee.mobile-left-align.center .action-area,
+.hero-marquee.mobile-left-align.center .lockup-area,
+.hero-marquee.mobile-left-align.center .row-qrcode .row-wrapper {
```

Nit: maybe `:is` would make this shorter:
```suggestion
.hero-marquee.mobile-left-align.center :is(.action-area, .lockup-area, .row-qrcode .row-wrapper) {
```
Same for L558-560